tex/chapters/survey/main.tex

```diff
@@ -1,6 +1,71 @@
 \chapter{Data collection}
 
 \section{Background on survey design}
+
+%cite: https://books.google.co.uk/books?id=mSRTDwAAQBAJ&lpg=PP1&ots=nynKGurXVG&dq=how%20to%20write%20a%20survey&lr&pg=PP1#v=onepage&q&f=false
+%cite: https://www.idsurvey.com/en/advantages-and-disadvantages-of-phone-survey/
```

Comment:
```suggestion
% TODO cite: https://books.google.co.uk/books?id=mSRTDwAAQBAJ&lpg=PP1&ots=nynKGurXVG&dq=how%20to%20write%20a%20survey&lr&pg=PP1#v=onepage&q&f=false
% TODO cite: https://www.idsurvey.com/en/advantages-and-disadvantages-of-phone-survey/
```

`# TODO` or `% TODO` or etc... Is a common convention for this sort of stuff (people often run commands to check that they can't find any files with any remaining `TODO`s left) :).

Also, FYI this is a GitHub "suggestion" you can directly commit the change on GitHub (you'll need to pull it down locally) which is a nice way of small little things getting fixed as I can show you exactly what I mean as opposed to describing it.

That doesn't mean you *have* to commit the suggestion: just that you can.

It's also possible to batch accept multiple suggestions at a time which helps keep the commits nice and clear :)